ppb/vfs.py

```diff
@@ -0,0 +1,97 @@
+"""
+Handles opening files from the Python "VFS".
+"""
+import logging
+from pathlib import Path
+import sys
+try:
+    import importlib.resources as impres
+except ImportError:
+    # Backport
```

Comment:
Please specify in a comment which Python versions require that.  (Presumably, only 3.6?)